demos/tab-scroller.html

```diff
@@ -0,0 +1,180 @@
+            <a href="/" class="mdc-toolbar__menu-icon"><i class="material-icons">&#xE5C4;</i></a>
+          </span>
+          <span class="mdc-toolbar__title catalog-title">Tab Scroller</span>
+        </section>
+      </div>
+    </header>
+
+    <main class="mdc-toolbar-fixed-adjust">
+      <section class="hero">
+        <div class="demo">
+          <div class="mdc-tab-scroller demo-scroller">
+            <div class="mdc-tab-scroller__content demo-content">
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+              <span class="demo-cube"></span>
+              <span class="demo-cube demo-cube--alt"></span>
+            </div>
+          </div>
+          <form class="demo-controller">
+            <div class="demo-controller-row">
+              <label for="id0">Scroll to</label>
+              <input id="id0" class="demo-scroll-value" type="number" name="scrollLeft" value="0">
+              <button type="submit">Scroll</button>
+            </div>
+            <div class="demo-controller-row">
+              <button type="button" class="demo-decrement">-50</button>
+              <button type="button" class="demo-increment">+50</button>
+            </div>
+          </form>
+        </div>
+      </section>
+
+      <section class="example">
+        <h2>Paging</h2>
+        <p>The paging scroller does not allow native scrolling inside the area.</p>
```

Comment:
I'm seemingly able to natively scroll within this, despite what this sentence says? AFAICT nothing references this CSS class or the JS constant for it.